modules/oe_content_extra_project/config/install/field.field.node.oe_project.oe_cx_project_lead_contributors.yml

```
@@ -0,0 +1,33 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_cx_project_lead_contributors
+    - node.type.oe_project
+    - oe_content_entity_organisation.oe_organisation_type.oe_stakeholder
+  module:
+    - composite_reference
+    - entity_reference_revisions
+third_party_settings:
+  composite_reference:
+    composite: true
+id: node.oe_project.oe_cx_project_lead_contributors
```

```suggestion
id: node.oe_project.oe_cx_lead_contributors
```
To be consistent, with the cx prefix remove the project prefix